presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -726,43 +726,43 @@ public double getCumulativeTotalMemory()
     @JsonProperty
     public DataSize getUserMemoryReservation()
     {
-        return userMemoryReservation;
+        return succinctBytes(userMemoryReservationInBytes);
```

Comment:
Why not change the method signature to return bytes ? some dependencies outside of this codebase ?

succinctBytes(...) --> this will create a new DataSize object everytime its called. So i am wondering if this change will really help or not .

Reply:
Discussed offline. This commit alone will not give any wins, we need to change the return types to long as well. So Shang agreed to work on the remaining api conversion stuff in a new commit on top of this exisiting commit.